contracts/oracle/implementations/OracleUSDCEURChainlink.sol

```diff
@@ -15,7 +15,7 @@ contract OracleUSDCEURChainlink is OracleChainlinkMultiEfficient {
         OracleChainlinkMultiEfficient(_stalePeriod, guardians)
     {}
 
-    function _circuitChainlink() internal pure override returns (AggregatorV3Interface[2] memory) {
+    function circuitChainlink() public pure override returns (AggregatorV3Interface[2] memory) {
```

Comment:
actually this need to be changed as I am doing in the borrow staked contracts. With undefined sizes

Reply:
Ah shit, has it been fixed in angle-solidity?